### PR TITLE
Small Tutorial Fixes

### PR DESCRIPTION
**Pull request by manunamz**
Add some clarifying code-comment padding (`// ...`) to show that the given code snippet is not the only code in the file (didn't get all of them though). 

Also fixed some typos.

Awesome tutorial. 🎉

### CODE REVIEW

File: docs/tutorial/saving-data.md [hunk 1]
```diff
@@ -216,13 +223,15 @@ create({
 })
 ```
 
-If you'll recall `<Form>` gives us all of the fields in a nice object where the key is the name of the field, which means the `data` object we're receiving in `onSubmit` is already in the proper format that we need for the `input`!
+If you'll [recall](https://learn-redwood.netlify.app/docs/tutorial/everyones-favorite-thing-to-build-forms/#onsubmit) `<Form>` gives us all of the fields in a nice object where the key is the name of the field, which means the `data` object we're receiving in `onSubmit` is already in the proper format that we need for the `input`!
```
Comment (clairefro) [inline docs/tutorial/saving-data.md:226]: We need to use relative links in order not to break links in i18n docs
```suggestion
If you'll [recall](/everyones-favorite-thing-to-build-forms/#onsubmit) `<Form>` gives us all of the fields in a nice object where the key is the name of the field, which means the `data` object we're receiving in `onSubmit` is already in the proper format that we need for the `input`!
```

**Review by clairefro — COMMENTED** [review]
Looks good thanks! Can you please change the absolute link to relative as suggested, in order not to break the multi-lingual docs?

**Comment by clairefro** [comment]
Finger slipped... meant to "request changes" but it went to comment instead! Once the link is fixed I will approve :)

**Comment by clairefro** [comment]
Just a reminder on this - I'm happy to merge once the above changes are addressed!

**Comment by manunamz** [comment]
Ack, sorry about the wait -- not sure how to add a single commit so I'll close this PR and open a new one.

**Pull request closed without merging**